metric/bigtable/src/main/java/com/spotify/heroic/metric/bigtable/api/Mutations.java

```diff
@@ -66,6 +68,7 @@ public Builder setCell(
                     .newBuilder()
                     .setFamilyName(family)
                     .setColumnQualifier(columnQualifier)
+                    .setTimestampMicros(TimeUnit.MILLISECONDS.toMicros(DateTime.now().getMillis()))
```

Comment:
Could also set this to `-1` to use the bigtable server time - https://cloud.google.com/bigtable/docs/reference/data/rpc/google.bigtable.v2#google.bigtable.v2.Mutation.SetCell

Reply:
good catch. changed